x/shared/session.go

```diff
@@ -118,8 +122,8 @@ func GetEarliestSupplierClaimCommitHeight(
 	// window open block hash and the supplier address.
 	randomNumber := poktrand.SeededInt63(claimWindowOpenBlockHash, []byte(supplierAddr))
 
-	distributionWindowSizeBlocks := sharedParams.GetClaimWindowCloseOffsetBlocks()
-	randCreateClaimHeightOffset := randomNumber % int64(distributionWindowSizeBlocks)
+	claimWindowNumBlocks := sharedParams.GetClaimWindowCloseOffsetBlocks() - closeBlockOffset
+	randCreateClaimHeightOffset := randomNumber % int64(claimWindowNumBlocks)
 
 	return claimWindowOpenHeight + randCreateClaimHeightOffset
```

Comment:
```suggestion
	// TODO_POST_MAINNET: Determine if we need to enforce claim distribution in order to avoid
	// all claims or proofs being accidently submitted at the same time.
	// claimWindowNumBlocks := sharedParams.GetClaimWindowCloseOffsetBlocks() - closeBlockOffset
	// randCreateClaimHeightOffset := randomNumber % int64(claimWindowNumBlocks)
	// return claimWindowOpenHeight + randCreateClaimHeightOffset
	return claimWindowOpenHeight
```